polyply/src/gen_coords.py

```diff
@@ -254,11 +255,15 @@ def gen_coords(toppath,
             LOGGER.warning(msg,  type="warning")
 
     # do a sanity check
-    LOGGER.info("checking residue integrity",  type="step")
-    check_residue_equivalence(topology)
+    if skip_filter:
```

Comment:
`if not skip_filter`?

Reply:
this is not the filter; if you skip the filtering this function ensures all residues are unique